enhancements/hide_container_mountpoints.md

```diff
@@ -0,0 +1,139 @@
+---
+title: hide-container-mountpoints
+authors:
+  - "@lack"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-01-18
+last-updated: 2021-01-18
+status: implementable
+---
+
+# Hide Container Mountponts
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The current implementation of kubelet and cri-o both use the top-level
+namespace for all container and kubelet mountpoints. However, moving these
+container-specific mountpoints into a private namespace reduced systemd
+overhead with no difference in functionality.
+
+## Motivation
+
+Systemd scans and re-scans mountpoints many times, adding a lot to the CPU
+utilization of systemd and overall overhead of the host OS running OpenShift.
+Changing systemd to reduce its scanning overhead is tracked in [BZ
+1819868](https://bugzilla.redhat.com/show_bug.cgi?id=1819868), but we can work
+around this exclusively within openshift. Using a separate mountpoint to for
+both crio-o and kubelet mounts can completely segregate all container-specific
```

Comment:
nit: s/crio-o/CRI-O

in general, for docs like this, we should use CRI-O (and I believe Kubelet should be capitalized as well)

Reply:
also all mentions of kubernetes and openshift should be Kubernetes and Openshift, I think